tests/ZendService/Amazon/Ec2/AbstractTest.php

```diff
@@ -32,7 +33,7 @@ public function testSetRegion()
     public function testSetInvalidRegionThrowsException()
     {
         $ec2 = new TestAmazonAbstract('TestAccessKey', 'TestSecretKey');
-        $this->setExpectedException(
+        $this->expectException(
             'ZendService\Amazon\Ec2\Exception\InvalidArgumentException',
```

Comment:
Strings with class names should be replaced with ::class and imported with `use` statements across all codebase for the benefit of static analysis tools.
Might be done as separate QA related PR along with coding styles fixes.